src/app/static/src/app/store/root/actions.ts

```diff
@@ -0,0 +1,19 @@
+import {ActionContext, ActionTree, Store} from "vuex";
+import {RootState} from "../../root";
+import {StepDescription} from "../stepper/stepper";
+
+export interface RootActions {
+    reload: (store: ActionContext<RootState, RootState>) => void;
+}
+export const actions: ActionTree<RootState, RootState> & RootActions = {
+
+    reload(store) {
+        const {state, getters, commit} = store;
+        const invalidSteps = state.stepper.steps.map((s: StepDescription) => s.number)
+            .filter((i: number) => i < state.stepper!!.activeStep && !getters["stepper/complete"][i]);
+
+        if (invalidSteps.length > 0) {
+            commit({type: "Reset"});
```

Comment:
The flow I've gone with is;

1. initialise store states from local storage
2. wait for async data to be ready
3. this action fires which resets the state if in an invalid state

Reply:
So this action isn't actually reloading anything, but should only be invoked by the stepper once the filles have been loaded from the backend (and the state should already have been initialised from local storage), and then this method checks that everything's good to go? Maybe it should be called 'validate' or something similar?